<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>replaceDialog</class>
 <widget class="QDialog" name="replaceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>查找</string>
  </property>
  <widget class="QPlainTextEdit" name="findContent">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>25</y>
     <width>221</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>查找内容(N):</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_capital">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>区分大小写</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_repeat">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>循环</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="title">
    <string>方向</string>
   </property>
   <widget class="QRadioButton" name="redioButton_down">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>18</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>向下</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="redioButton_up">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>18</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>向上</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="button_findNext">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>25</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>查找下一个(&amp;F)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_cancel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>60</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
